jenkinsfiles/ginkgo-kubernetes-all.Jenkinsfile

```diff
@@ -39,7 +39,8 @@ pipeline {
         SERVER_BOX = "cilium/ubuntu"
         FAILFAST=setIfLabel("ci/fail-fast", "true", "false")
         CNI_INTEGRATION=setIfLabel("integration/cni-flannel", "FLANNEL", "")
-        GINKGO_TIMEOUT="98m"
+        HOST_FIREWALL=setIfLabel("ci/host-firewall", "1", "0")
+        GINKGO_TIMEOUT="118m"
```

Comment:
I think this line was committed by mistake (probably due to merge conflict).

Reply:
Yes, I had to resolve merge conflicts. So should I revert this to `GINKGO_TIMEOUT="98m"`? Looks like the timeout was changed in some other PR that's not backported to 1.9.

Reply:
We can either label and backport that other PR or revert to `GINKGO_TIMEOUT="98m"`. I don't think we've hit the timeout issue on v1.9, but maybe it's just a matter of time.

Reply:
/cc @nebril I've reverted the value.